<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeSelect</class>
 <widget class="QDialog" name="TimeSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>233</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>211</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>起始时间</string>
   </property>
   <widget class="QDateTimeEdit" name="fromDateTimeEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd H:mm:ss</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>211</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>截止时间</string>
   </property>
   <widget class="QDateTimeEdit" name="endDateTimeEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd H:mm:ss</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="okBtn">
      <property name="text">
       <string>确定(&amp;O)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelBtn">
      <property name="text">
       <string>取消(&amp;C)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
